app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,57 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      include Response
+      include ExceptionHandler
+
+      def show
+        json_response(find_volunteer)
+      end
+
+      def create
+        json_response(Volunteer.create!(volunteer_params))
+      end
+
+      def update
+        json_response(find_volunteer.update!(volunteer_params))
+      end
+
+      def destroy
+        find_volunteer.destroy!
```

Comment:
You don't need `.destroy!` destroy is fine. `.find!` already raises an error if the object cannot be found